### PR TITLE
DMP-184 Add event type records to database

### PR DESCRIPTION
**Pull request by richardbull**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-184

### Change description ###

Add 376 rows in postgres (darts-db) moj_event_type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java [hunk 1]
```diff
@@ -20,8 +23,8 @@ public class Courthouse {
 
     @Id
     @Column(name = "moj_crt_id")
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "courthouse_gen")
-    @SequenceGenerator(name = "courthouse_gen", sequenceName = "moj_crt_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_cth_gen")
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java:26]: should the sequence generator not match the name of the primary key? i.e. moj_crt_gen ?

File: src/main/java/uk/gov/hmcts/darts/common/entity/Case.java [hunk 2]
```diff
@@ -5,76 +5,64 @@
...
 @SuppressWarnings({"PMD.ShortClassName"})
 public class Case {
 
     private static final String MAPPED_BY_THE_CASE = "theCase";
 
     @Id
     @Column(name = "moj_cas_id")
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_cas_gen")
+    @SequenceGenerator(name = "moj_cas_gen", sequenceName = "moj_cas_seq", allocationSize = 1)
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/common/entity/Case.java:31]: i don't think we should have an allocationSize of 1

File: src/main/java/uk/gov/hmcts/darts/common/entity/Event.java [hunk 3]
```diff
@@ -18,9 +19,16 @@ public class Event {
 
     @Id
     @Column(name = "moj_eve_id")
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_eve_gen")
+    @SequenceGenerator(name = "moj_eve_gen", sequenceName = "moj_eve_seq", allocationSize = 1)
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/common/entity/Event.java:23]: i don't think allocationSize should be this low.

File: src/main/java/uk/gov/hmcts/darts/common/entity/Case.java (hunk 2 above)
Comment (richardbull) [reply src/main/java/uk/gov/hmcts/darts/common/entity/Case.java:31]: Need to leave at 1 for now to match our sql. Barry is going to review the proposed schema and will update his scripts with some sequences having a modified `increment` and `cache` options and we can change our entities to match after that.

**Review by cmagowan-hmcts — APPROVED** [review]

**Pull request merged**